web-app/src/app/app.component.css

```diff
@@ -67,23 +67,25 @@ and (max-width: 992px){
     margin-right:5em;
   }
 }
+
+@media only screen
+and (min-width: 992px)
+and (max-width: 1200px){
```

Comment:
In theory, if their screen was wider than `1200px`, we still want to not-display it, right?